src/main/java/com/monitorjbl/xlsx/impl/StreamingCell.java

```diff
@@ -146,6 +246,13 @@ public Date getDateCellValue() {
   /* Not supported */
 
   /**
+   * Not supported.  Added for compatability with POI 3.12.
+   */
+  public void removeHyperlink() {
```

Comment:
as indicated in the JavaDoc, I am using POI 3.12 (the latest stable version).  The code will not compile without implementing this Row method (https://poi.apache.org/apidocs/org/apache/poi/ss/usermodel/Cell.html#removeHyperlink()).